azure-pipelines/templates/test.yml

```diff
@@ -0,0 +1,35 @@
+jobs:
+    - job: UnitTests
+
+      strategy:
+          matrix:
+              Node14:
+                  NODE_VERSION: '14.x'
```

Comment:
Do we want still want to test 14.x?

Reply:
Node 14 and 16 are supported by us until June 30th. Docs [here](https://learn.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=isolated-process%2Cv4&pivots=programming-language-javascript)

For this PR I didn't want to change any of that, but I figure sometime soon we'll go through everything to remove v14/v16 and add v22